torch_xla/csrc/tensor_util.h

```diff
@@ -146,8 +146,50 @@ inline bool tensor_has_dym_dim(at::Tensor t) {
   return !c10::asIntArrayRefSlowOpt(sym_sizes).has_value();
 }
 
+inline bool tensors_has_dym_dim(at::TensorList tensors) {
+  for (const at::Tensor t : tensors) {
+    if (tensor_has_dym_dim(t)) {
+      return true;
+    }
+  }
+  return false;
+}
+
+inline std::vector<at::Tensor> xla_expand_outplace_symint(
```

Comment:
I think it is better to call this function `xla_expand_outplace_symint_helper`  and you create another `xla_expand_outplace_helper` which handle the static case. In `xla_expand_outplace` it will have a if check to dispatch to one of the helper.

Reply:
fixed